### ISSUE TITLE
Excluded option on a field should support filters

### ISSUE THREAD

**Issue by nghuuphuoc** [formvalidation/support#62]
_From @phlegx on March 10, 2015 13:40_

The ```excluded``` option on a field can only be of type boolean (v0.6.1). It can be very useful if the option can also be of type filter like in the ```excluded``` option of a form. Here an example:

```javascript
    ...
      fields: {
        'field_one': {
          excluded: false
        },
        'field_two': {
          excluded: function(field, validator) {
            return !field.closest('fieldset').is(':visible');
          }
        }
     }
```

_Copied from original issue: formvalidation/formvalidation#1453_

**Comment by nghuuphuoc** [comment]
@phlegx
Thank for the suggestion! This feature is implemented and will be available in the next v0.6.2.

You can use in 3 ways:

1)
```javascript
$('#excludedForm').formValidation({
    fields: {
        username: {
            excluded: function($field, validator) {
                return true;
            }
        }
    }
});
```

2)
```javascript
function excludeField($field, validator) {
    return ($field.attr('name') === 'username')
                                ? false     // Don't exclude the username field
                                : true;     // Exclude the email field
}

$('#excludedForm').formValidation({
    fields: {
        username: {
            excluded: 'excludeField'
        },
        email: {
            excluded: 'excludeField'
        }
    }
});
```

3) Declarative way:

```javascript
function excludeField($field, validator) {
    return ($field.attr('name') === 'username')
                                ? false     // Don't exclude the username field
                                : true;     // Exclude the email field
}

<input type="text" name="username" required data-fv-excluded="excludeField" />

<input type="text" name="email" required data-fv-emailaddress data-fv-excluded="excludeField" />
```

**Comment by phlegx** [comment]
:+1:  Thx @nghuuphuoc. Nice work!
